azure-functions/acbs-function/acbs-amend-facility-loan-record/index.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Facility Loan Record amendment SOF
+ * **********************************
+ * This is a sub-orchestrated function invoked from it's master orchestrator.
+ * `acbs-amend-facility` is the master orchestrator for this function.
+ *
+ * Sub-Orchestration Function (SOF)
+ * --------------------------------
+ * This function cannot be invoked directly and must be invoked by a DOF.
+ *
+ * Prerequisites
+ * -------------
+ * 0. 'npm install durable-functions'
+ * 1. Durable HTTP starter function (acbs-http)
+ * 2. DOF (acbs-amend-facility)
+ * 3. DAF (activity-get-loan-id, activity-update-facility-loan-amount, activity-update-facility-loan)
+ *
+ * ACBS
+ * ----
+ * This function is responsible for amending FLR.
+ * FLR is one of a child record of an issued facility.
+ */
+const df = require('durable-functions');
+const retryOptions = require('../helpers/retryOptions');
+const mappings = require('../mappings');
+const CONSTANTS = require('../constants');
```

Comment:
If we only need the Facility, then we can destructure it here

Reply:
Good spot, have updated.